index.html

```diff
@@ -0,0 +1,1546 @@
+                Homes 2021. If you work in this field, please participate in
+                our survey.
+              </p>
+            </div>
+          </div>
+          <div class="card">
+            <div class="feature-img-div">
+              <img class="card-img-top"
+                src="https://d.newsweek.com/en/full/1612630/sweet-james-accident-attorneys-v2.jpg"
+                alt="Card image cap" />
+              <div class="feature-tag blue">SPONSORED INSIGHT</div>
+            </div>
+            <div class="card-body ranking-border">
+              <h5 class="card-title fam-hd">
+                America's Best Addiction Treatment Centers 2020
+              </h5>
+              <p class="card-text font-15">
+                Newsweek partnered with global market research firm Statista
+                Inc. to rank the best U.S. treatment facilities that focus on
+                addiction.
+              </p>
+            </div>
+          </div>
+          <div class="card">
+            <div class="feature-img-div">
+              <img class="card-img-top"
+                src="https://d.newsweek.com/en/full/1621635/americas-best-addiction-treatment-centers-2020.png"
+                alt="Card image cap" />
+              <div class="feature-tag blue">SPONSORED INSIGHT</div>
+            </div>
+            <div class="card-body ranking-border">
+              <h5 class="card-title fam-hd">
+                What Makes the Best Lawyers Stand Out?
+              </h5>
+              <p class="card-text font-15">
+                The unthinkable has happened and you need a good lawyer to
+                turn to, how do you chose?
+              </p>
+            </div>
+          </div>
+          <div class="card">
+            <div class="feature-img-div">
+              <img class="card-img-top"
+                src="https://d.newsweek.com/en/full/1587568/infection-prevention-products-banner.jpg"
+                alt="Card image cap" />
+              <div class="feature-tag blue">SPONSORED INSIGHT</div>
+            </div>
+            <div class="card-body ranking-border">
+              <h5 class="card-title fam-hd">
+                Best Infection Prevention Products
+              </h5>
+              <p class="card-text font-15">
+                During the coronavirus outbreak, health care is more important
+                than ever. In Newsweek's first Best Health Care series, we
+                highlight the best infection prevention products.
+              </p>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="row align-items-center justify-content-center section-topic">
+      <!-- <div class="col-"> -->
+      <span class="n-log">N</span>
+      <h2 class="topic-h2 mb-0">IN THE MAGAZINE</h2>
+      <!-- </div> -->
+    </div>
+
+    <!-- *** The "In The Magazine" section starts here *** -->
+    <div class="row in-the-magazine">
+      <!-- <div class="row"> -->
```

Comment:
- [ ] Please delete **commented piece of codes** in your HTML and CSS files. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))  
:x: **commented piece of codes** /* `.nav { color: red;}` */  
:heavy_check_mark: **comments about code** /*Navigation */